homeassistant/components/freebox/home_base.py

```diff
@@ -77,19 +77,30 @@ async def async_update_signal(self):
             )
         self.async_write_ha_state()
 
-    async def set_home_endpoint_value(self, command_id: Any, value=None) -> None:
+    async def set_home_endpoint_value(self, command_id: Any, value=None) -> bool:
         """Set Home endpoint value."""
         if command_id is None:
             _LOGGER.error("Unable to SET a value through the API. Command is None")
-            return
+            return False
+
         await self._router.home.set_home_endpoint_value(
             self._id, command_id, {"value": value}
         )
+        return True
```

Comment:
```suggestion
            return None

        return await self._router.home.set_home_endpoint_value(
            self._id, command_id, {"value": value}
        )
```
What do you think ?

Reply:
hum the function's result is used to check if the value is set or not. I'm not sure to modify it

Reply:
If `set_home_endpoint_value` returns None or False, yes, better check the return type before